### PR TITLE
Inactives language fix

### LINKED ISSUE #833: Find Inactives doesn't work
I was reviewing this extension for 15 minutes and never gave me a result. 
To others could be added the option to search inactive blog 1 week pls

I follow only 48 blogs and 15 minutes for 48 blogs is so much or not? :cry: 

screenshot_44

PS: Excuse my English, I speak Spanish but I try to help.

### PR DESCRIPTION
**Pull request by ChuckL**
Fixed #833 

If the user had a different language than English it would fail.

### CODE REVIEW

File: Extensions/find_inactives.js [hunk 1]
```diff
@@ -68,8 +75,11 @@ XKit.extensions.find_inactives = new Object({
 	},
 
 	get_count: function() {
-		var count_text = $("#tabs").html().match(/Following.*Tumblr/gm)[0];
-		count_text = count_text.replace(/[^0-9\.]/g, '');
+		var count_text = $("#tabs").html();
+		count_text = count_text.replace(/[^0-9]/g, '');
+
+		console.log("User count: " + count_text);
```
Comment (bvtsang) [inline Extensions/find_inactives.js]: We can leave this out.

File: Extensions/find_inactives.js [hunk 2]
```diff
@@ -165,19 +173,42 @@ XKit.extensions.find_inactives = new Object({
...
-		if(updated_time.indexOf("day") > -1){
-			return time_value;
+		for(var i = 0; i < this.language_date_strip.length; i++) {
+			updated_time = updated_time.replace(this.language_date_strip[i], '');
 		}
-		if(updated_time.indexOf("week") > -1){
-			return time_value * 7;
+
+
+		var days = this.calculate_days(updated_time, time_value, this.language_date_day, 1);
```
Comment (bvtsang) [inline Extensions/find_inactives.js]: This `days_since_updated` function has a lot of duplicate code that can be refactored.

Comment (nightpool) [inline Extensions/find_inactives.js]: I'm fine with doing the simplest possible thing for now. Its not like we're going to need to touch this code a lot. (or change it in a way that refactoring will help with)

File: Extensions/find_inactives.js [hunk 3]
```diff
@@ -68,8 +75,10 @@ XKit.extensions.find_inactives = new Object({
 	},
 
 	get_count: function() {
-		var count_text = $("#tabs").html().match(/Following.*Tumblr/gm)[0];
-		count_text = count_text.replace(/[^0-9\.]/g, '');
+		var count_text = $("#tabs").html();
+		count_text = count_text.replace(/[^0-9]/g, '');
```
Comment (nightpool) [inline Extensions/find_inactives.js]: would prefer to do peopleCount = parseInt(count_text.replace(etc)) so that the operations are selfdocumenting

**Comment by ChuckL** [comment]
@nightpool I think it's fine, it's one logical step per line rather than nested method calls.
I could make another variable to make it easier to read / more documentation if you think that's better.

@bvtsang I opted to have some duplicated code since I think it's easier to read and understand than what I would do probably do otherwise.  Make an enum type object and loop through the enums.

**Comment by nightpool** [comment]
@ChuckL but turning it from the string into the int *is* a logical operation, right? and that requires dealing both with the useless text and the parseInt function.

I agree with keeping the duplicated code easier to read btw @bvtsang @ChuckL

**Comment by ChuckL** [comment]
Turning a string to an int in a logical operation.

destroying useless text text then turning it into an interger, in my mind isn't a single logical operation.

**Comment by nightpool** [comment]
down for whatever tbh. its a minor nitpick

**Comment by ChuckL** [comment]
So is this good to go or are there a few bugs / styling stuff still?

**Comment by hobinjk** [comment]
Tested locally, LGTM. I don't think any of the remaining nitpicks are blockers.
@homu r+

**Pull request merged**